app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -472,12 +477,19 @@ export default {
     saveDraft(conversationId, replyType) {
       if (this.message) {
         const savedDraftMessages = this.getSavedDraftMessages();
-        LocalStorage.set(LOCAL_STORAGE_KEYS.DRAFT_MESSAGES, {
-          ...savedDraftMessages,
-          [`draft-${conversationId}-${replyType}`]: trimMessage(
-            this.message || ''
-          ),
-        });
+        const key = `draft-${conversationId}-${replyType}`;
+        const draftToSave = trimMessage(this.message || '');
+
+        const currentDraft = savedDraftMessages[key];
+
+        if (currentDraft !== draftToSave) {
+          LocalStorage.set(LOCAL_STORAGE_KEYS.DRAFT_MESSAGES, {
+            ...savedDraftMessages,
+            [key]: draftToSave,
+          });
+
+          this.showAlert(this.$t('CONVERSATION.DRAFT_SAVED'));
```

Comment:
Please remove the draft updated message. Getting an alert every 10s while typing a long message is not a pleasant sight.

Reply:
It would save an unwanted update + operation on that object. The object size could be large as app usage increases. 
This is neccessary IMHO

Reply:
I don't think it would make much of difference here as the character count are max 10000 chars.